addons/account/models/ir_module.py

```diff
@@ -61,13 +61,14 @@ def _compute_account_templates(self):
     def write(self, vals):
         # Instanciate the first template of the module on the current company upon installing the module
         was_installed = len(self) == 1 and self.state == 'installed'
-        super().write(vals)
+        res = super().write(vals)
         is_installed = len(self) == 1 and self.state == 'installed'
-        if not was_installed and is_installed and not self.env.company.chart_template and self.account_templates:
+        if len(self) == 1 and not was_installed and is_installed and not self.env.company.chart_template and self.account_templates:
```

Comment:
```suggestion
        if not was_installed and is_installed and not self.env.company.chart_template and self.account_templates:
```
part of both `was_installed` and `is_installed` already

Reply:
Not really, it'll try loading the chart template and crash when self is empty

Reply:
I don't get it, can you elaborate?